ankihub/gui/enable_fsrs_dialog.py

```diff
@@ -0,0 +1,84 @@
+from datetime import date
+from typing import Optional
+
+import aqt
+
+from .. import LOGGER
+from ..settings import ANKI_INT_VERSION, config
+from .deck_options import MIN_ANKI_VERSION_FOR_FSRS_FEATURES
+from .utils import show_dialog
+
+ENABLE_FSRS_REMINDER_INTERVAL = 30
+
+
+def maybe_show_enable_fsrs_reminder():
+    if ANKI_INT_VERSION < MIN_ANKI_VERSION_FOR_FSRS_FEATURES:
+        return
+
+    if not config.get_feature_flags().get("fsrs_reminder", False):
+        return
+
+    anking_deck_id = config.anking_deck_id
+    if not bool(config.deck_config(anking_deck_id)):
+        return
+
+    if aqt.mw.col.get_config("fsrs"):
+        return
+
+    if not config._private_config.show_enable_fsrs_remind_again:
+        return
+
+    last_remind_date = config.get_days_since_last_enable_fsrs_reminder()
+    today_date = date.today()
+    if last_remind_date is None or last_remind_date >= ENABLE_FSRS_REMINDER_INTERVAL:
+        config.set_last_enable_fsrs_reminder_date(today_date)
+        _show_enable_fsrs_reminder()
+
+
+def _show_enable_fsrs_reminder() -> None:
+    def on_button_clicked(button_idx: Optional[int]) -> None:
+        enable = button_idx == 1
+
+        dont_show_again = dialog.checkbox.isChecked()
+
+        LOGGER.info(
+            "enable_fsrs_reminder_dialog_choice",
+            user_choice="enable" if enable else "skip",
+            dont_show_again=dont_show_again,
+        )
+
+        if dont_show_again:
+            config.set_show_enable_fsrs_remind_again(False)
+            return
```

Comment:
:wrench: If the user clicks "Enable", we should enable FSRS, even if they also check the "Don't show again" checkbox
```suggestion
```